src/code/containers/modal-message-container.js

```diff
@@ -2,7 +2,8 @@ import { connect } from 'react-redux';
 import ModalAlert from '../components/modal-alert';
 import * as actions from '../actions';
 import { resetGametes } from '../modules/gametes';
-Object.assign(actions, { resetGametes });
+// merge with the other actions
+actions.resetGametes = resetGametes;
```

Comment:
My impression was that the purpose of the polyfill was to make it work in IE11, but I must misunderstand. We can catch up about it offline